sources/dfmc/reader/interface.dylan

```diff
@@ -109,6 +109,11 @@ define serious-program-warning <ratios-not-supported> (<invalid-token>)
   format-arguments token-string;
 end serious-program-warning;
 
+define serious-program-warning <unterminated-hash-literal>
+  format-string "Unterminated hash literal %s";
```

Comment:
"hash literal" is vague terminology and doesn't match the documentation.

Reply:
Done